HW_Lesson_4/HW_L4_Task_2.py

```diff
@@ -0,0 +1,17 @@
+lst = ["FirstItem", "FriendsList", "MyTuple"]
+res = []
+
+for word in lst:
+    word = word.lower()
+    if word[-4:] == "item" or word[-4:] == "list":
+        word = list(word)
+        word.insert(-4, "_")
+        res.append("".join(word))
+    elif word[-5:] == "tuple":
+        word = list(word)
+        word.insert(-5, "_")
+        res.append("".join(word))
+    else:
+        res.append(word)
+
+print(res)
```

Comment:
your code should work with any list of data. 
["FirstItem", "FriendsList", "MyTuple", 'AnotherWord'] for this one I got ['first_item', 'friends_list', 'my_tuple', 'anotherword']

Reply:
["FirstItem", "FriendsList", "MyTuple", "AnotherWordfromougterscope", "word", "Another_dict"]

result :
['first_item', 'friends_list', 'my_tuple', 'anotherwordfromougters_cope', 'word', 'another_dict']


hardcode indexes in your code is not a good idea)) better to use regex

Reply:
Sporting interest, should have been done without regular expressions already)